packages/components/src/components/Table/TableRow.tsx

```diff
@@ -68,21 +68,31 @@ export const Row = styled.tr<{
 export interface TableRowProps {
     children: ReactNode;
     isCollapsed?: boolean;
+    highlightRowOnHover?: boolean;
     onClick?: () => void;
+    onHover?: (isHovering: boolean) => void;
 }
 
-export const TableRow = ({ children, isCollapsed = false, onClick }: TableRowProps) => {
+export const TableRow = ({
+    children,
+    isCollapsed = false,
+    onClick,
+    onHover,
+    highlightRowOnHover,
+}: TableRowProps) => {
     const { elevation } = useElevation();
     const isHeader = useTableHeader();
-    const { highlightRowOnHover } = useTable();
+    const { highlightRowOnHover: highlightRowOnHoverGlobal } = useTable();
 
     return (
         <Row
             $elevation={elevation}
             $isCollapsed={isCollapsed}
-            $highlight={highlightRowOnHover}
+            $highlight={highlightRowOnHover ?? highlightRowOnHoverGlobal}
```

Comment:
What about `$isHighlighted`?

It looks a bit inconsistent when you have `$isHeader` and `$isCollapsed` with `is` here